webcompat/form.py

```diff
@@ -48,32 +52,38 @@
 username_message = u'A valid username must be {0} characters long'.format(
     random.randrange(0, 99))
 
-problem_label = (u'What seems to be the trouble?',
-                 '<span class="wc-Form-required">*</span>')
-url_label = u'Site URL <span class="wc-Form-required">*</span>'
+desc_label = 'Please describe what was wrong <span class="wc-Form-required">' \
+    '*</span>'
+desc_message = u'An issue description is required.'
 
-desc_default = u'''1. Navigate to: Site URL
-2. …
+url_label = u'Site URL <span class="wc-Form-required">*</span>'
+browser_test_label = u'Did you test in another browser?'
 
-Expected Behavior:
+steps_default = u'''1. I've tried to log in.
```

Comment:
@zoepage @miketaylr the current placeholder format for the description details text area is breaking one functional test and seems to render one existing function obsolete.

https://github.com/webcompat/webcompat.com/blob/master/tests/functional/index-non-auth.js#L32

https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/js/lib/bugform.js#L503

is the copyURL() function now obsolete and I should remove it?